src/http.c

```diff
@@ -160,6 +160,20 @@ int callback_http(struct lws *wsi, enum lws_callback_reasons reason, void *user,
         break;
       }
 
+      // accessing `/base-path` redirects to `/base-path/`
+      if (strcmp(pss->path, endpoints.parent) == 0) {
+        if (lws_add_http_header_status(wsi, HTTP_STATUS_FOUND, &p, end)
+            || lws_add_http_header_by_name(wsi, "location:", endpoints.index,
```

Comment:
please use `lws_add_http_header_by_token` with `WSI_TOKEN_HTTP_LOCATION`

Reply:
Thanks, I somehow missed the `WSI_TOKEN_HTTP_LOCATION` when I looked for it. I've fixed it in the last commit.